automation/taskcluster/decisionlib.py

```diff
@@ -469,7 +470,7 @@ def with_dockerfile(self, dockerfile, use_indexed_task=True):
 
         image_build_task = (
             DockerWorkerTask("Docker image: " + image_name)
-            .with_worker_type(CONFIG.docker_image_build_worker_type or self.worker_type)
+            .with_worker_type(CONFIG.images_worker_type or self.worker_type)
```

Comment:
Wondering should I just trim the `or self.worker_type` part since we're alwasy going to default to one of the two types of workers, per level.

Reply:
Yeah that sounds safer. 👍

Reply:
Cool, done